osquery/sql/virtual_table.cpp

```diff
@@ -604,6 +609,11 @@ int xCreate(sqlite3* db,
         }
       }
 
+      if (is_extension && FLAGS_compat_index_all_extension_columns &&
+          0 == (int)options) {
```

Comment:
Instead of the c-style cast can you compare: `ColumnOptions::DEFAULT == options`.